docs/tutorial/fdfitting.rst

```diff
@@ -131,25 +113,22 @@ If we also expect an offset in the dependent parameter, we can simply add an off
     Name            Value  Unit      Fitted      Lower bound    Upper bound
     ------------  -------  --------  --------  -------------  -------------
     DNA/d_offset     0.01  [au]      True               -0.1            0.1
-    DNA/Lp          40     [nm]      True                0            100
-    DNA/Lc          16     [micron]  True                0            inf
-    DNA/St        1500     [pN]      True                0            inf
-    kT               4.11  [pN*nm]   False               0              8
+    DNA/Lp          40     [nm]      True              0.001            100
+    DNA/Lc          16     [micron]  True            0.00034            inf
+    DNA/St        1500     [pN]      True                  1            inf
+    kT               4.11  [pN*nm]   False              3.77              8
     DNA/f_offset     0.01  [pN]      True               -0.1            0.1
 
-From the above example, you can see how easy it is to composite models. Sometimes, models become more complicated. For
-instance, we may have two worm like chain models that we wish to add, and then invert. For the Odijk model, this can be
-done as follows::
+Sometimes, models become more complicated. For instance, we may have two worm like chain models, one for a DNA tether and the other for an unfolded protein. The total length of the construct is then the sum of the length of the DNA and the protein and the total distance is given by::
 
     model = lk.ewlc_odijk_distance("DNA") + lk.ewlc_odijk_distance("protein") + lk.distance_offset("offset")
     model = model.invert()
 
-Note how we added three models and then inverted the composition of those models. Models inverted via `invert()` will
-typically be slower than the pre-inverted counterparts. This is because the inversion is done numerically rather than
-analytically. For more complex examples on how this inversion may be used, please see the examples.
+Note how the three models are all defined as distance as a function of force. Since fitting is best done for force as a function of distance, we then inverted the composited model. Note that models inverted via :meth:`.invert() <lumicks.pylake.fitting.model.Model.invert()>` will
```

Comment:
```suggestion
Note how the three models all define distance as a function of force. Since fitting is best done for force as a function of distance, we then invert the composited model. Note that models inverted via :meth:`.invert() <lumicks.pylake.fitting.model.Model.invert()>` will
```